core/arch/arm/kernel/user_ta.c

```diff
@@ -625,16 +625,19 @@ TEE_Result tee_ta_init_user_ta_session(const TEE_UUID *uuid,
 			struct tee_ta_session *s)
 {
 	const struct user_ta_store_ops *store;
-	TEE_Result res = TEE_ERROR_ITEM_NOT_FOUND;
+	TEE_Result res;
```

Comment:
I though these changes where to go into 6ae578d "Add support for several user TA stores"